composeApp/src/commonMain/kotlin/org/ooni/probe/di/Dependencies.kt

```diff
@@ -312,6 +331,30 @@ class Dependencies(
             uploadMissingMeasurements = uploadMissingMeasurements::invoke,
         )
 
+    fun addDescriptorViewModel(
+        descriptorId: String,
+        onBack: () -> Unit,
+    ) = AddDescriptorViewModel(
+        onCancel = {
+            // TODO(aanorbel): show toast/snackbar
+            onBack()
+        },
+        onError = {
+            // TODO(aanorbel): show toast/snackbar
+            onBack()
+        },
+        saveTestDescriptors = {
+            saveTestDescriptors.invoke(it)
+            // TODO(aanorbel): show toast/snackbar
+            onBack()
```

Comment:
I would prefer if we don't have logic in the `Dependencies` class where it's hard to find. Why not call `onBack` inside of the viewModel?